src/braket/experimental/autoqasm/operators/return_statements.py

```diff
@@ -0,0 +1,52 @@
+# Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You
+# may not use this file except in compliance with the License. A copy of
+# the License is located at
+#
+#     http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+
+
+"""Operations to override return statement behavior."""
+
+from typing import Any
+
+from braket.circuits.free_parameter_expression import FreeParameterExpression
+from braket.experimental.autoqasm import program
+from braket.experimental.autoqasm import types as aq_types
+
+
+def return_output_from_main_(name: str, value: Any) -> Any:
```

Comment:
Ah, good call out. I was mindlessly following the convention from the conditional expressions and logical expressions that autograph must've used to avoid name conflicts with keywords (at least for the logical ops). I'll remove it